ocs_ci/ocs/resources/bucketclass.py

```
@@ -131,11 +133,23 @@ def _create_bucket_class(bucket_class_dict):
 
+            archive_policy = bucket_class_dict.get("archive_policy")
+            if "archive_nss_dict" in bucket_class_dict:
+                archive_nss = namespace_store_factory(
+                    interface, bucket_class_dict["archive_nss_dict"]
+                )
+                archive_policy = archive_nss[0].name
+
             if "vector_policy" in bucket_class_dict and interface.lower() != "oc":
                 raise RuntimeError(
                     f"vector_policy is supported only with 'oc' interface, got: {interface}"
                 )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)